test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/CollectionTest.scala

```diff
@@ -226,13 +226,100 @@ trait CollectionTest {
     assertEquals(5, coll.size())
     assertIteratorSameElementsAsSet(-45, 0, 12, 32, 42)(coll.iterator())
   }
+
+  @Test def toStringShouldConvertEmptyCollection(): Unit = {
+    val coll = factory.empty[Double]
+
+    val expected = "[]"
+    val result = coll.toString()
+    assertEquals(expected, result)
+  }
+
+  @Test def toStringShouldConvertOneElementCollection(): Unit = {
+    val coll = factory.fromElements[Double](1.01)
+
+    // Javascript displays n.0 as n, so one trailing digit must be non-zero.
+    val expected = "[1.01]"
+    val result = coll.toString()
+    assertEquals(expected, result)
+  }
+
+  @Test def toStringShouldUseCommaSpace(): Unit = {
+
+     // Choose Doubles which display the same in Java and Scala-js.
+    // Javascript displays n.0 as n, so one trailing digit must be non-zero.
+    val elements = Seq(88.42, -23.36, 60.173)
+
+    val coll = factory.fromElements[Double](elements:_*)
+
+    // The order of elements returned by each collection is defined
+    // by the collection. Be prepared to handle the general case of any
+    // order here. Specific collections should test the order they specify.
+    val answers = elements.permutations.map(_.mkString("[", ", ", "]"))
+
+    val expected = answers.toSet
+
+    val result = coll.toString()
+
+    val resultIsValid =  expected.contains(result)
+
+    val msg = "result '" + result +
+        "' not in expected set '" + expected + "'"
+
+    assertTrue(msg, resultIsValid)
```

Comment:
Here also, I guess the coding style of Scala Native (where IIUC you contribute quite a lot) is quite different from ours. I do like blank lines when they help structure things, but here I believe a denser style, combined with a few inlining of `val`s, can help readability:
```scala
val expected = elements.permutations.map(_.mkString("[", ", ", "]")).toSet
val result = coll.toString()
assertTrue(s"result '$result' not in expected set '$expected'",
    expected.contains(result)
```
The same applies to the other tests below.

Reply:
Changed as requested, in all tests.

One test has an exactly 80 character line for 'expected' because it is at an additional
indentation level.  Normally I would have put the part after the equal sign on the next
line, indented 2 spaces. CODINGSTYLE.md says this is OK. Here I left it at one line
because the line break was not absolutely necessary.